### PR TITLE
feat: some practice command shorthandlings

### PR DESCRIPTION
**Pull request by Satttoshi**
- Added and tested some practice mode shorthand commands
E.g
`.savenade` -> `.sn`
`.loadnade` -> `.ln`
`.deletenade` -> `.dn`
... and so on

- Added smart quering for nadenames, where the closest name is being selected for loading names
E.g

Ingame chat:
```
[MatchZy] Lineup omegalul loaded successfully!
[ALL] ●  yush‎: .ln ome
[MatchZy] Lineup useless-smoke loaded successfully!
[ALL] ●  yush‎: .ln use
```

- Added to allow the same lineup-name on different maps, so you can pick like `b-smoke` multiple times, but once per map. Updated logic for `savenade`, `deletenade`, `importnade` for this.

### CODE REVIEW

**Comment by Satttoshi** [comment]
Please wait with the review, until approved by me. I need to do some more testings.

**Comment by Satttoshi** [comment]
ready for review.

**Comment by mavproductions** [comment]
Love the idea of smart querying of util names. aliases for things like saving and deleting are always nice as well

**Comment by shobhit-pathak** [comment]
Thanks for the PR! It would be very helpful :D
Allow me some time for reviewing, I'll go through it asap

**Comment by Satttoshi** [comment]
Hey @shobhit-pathak I had a little though in general.
What do you think about setting up some unit testing, so we can use `dotnet test` and having a Pipeline checking PRs for integrity. For example I did test my smart querying function externally, but would be helpful if the project itself tests it for edge casing. Also my little refactoring did break the command arguments (in this case nade descriptions went missing), would be nice if a unit test would've told me: "Hey you did break the Command Args". Doesn't mean you should write test, but I would've liked to see a testing framework being setted up, which we can use for stuff like this :)

**Comment by shobhit-pathak** [comment]
Hey!

Yes having test cases would be very helpful for such cases, I will start exploring on how can we setup test cases for the plugin and how would it work, as right now I don't have any idea of it xd

Thanks for the suggestion and the PR!

**Review by shobhit-pathak — APPROVED** [review]

File: PracticeMode.cs [hunk 1]
```diff
@@ -55,6 +55,52 @@ public override int GetHashCode()
...
+            return bigrams;
+        }
+
+        /// <summary>
+        /// Finds the name from a list of names that is nearest to the input name using the Dice coefficient.
+        /// </summary>
+        /// <param name="inputName">The input name to match.</param>
+        /// <param name="names">The list of names to search from.</param>
+        /// <returns>The nearest matching name from the list.</returns>
+        public static string FindNearestName(string inputName, List<string> names) { }
```
Comment (Satttoshi) [inline PracticeMode.cs:87]: Umm. looks like something went wrong here, but it still works. Could you patch that real quick @shobhit-pathak ?

**Pull request merged**